<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="47" started-at="2019-11-06T07:36:03Z" finished-at="2019-11-06T07:36:03Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="47" started-at="2019-11-06T07:36:03Z" finished-at="2019-11-06T07:36:03Z">
      <class name="it.pkg.dynamic.Testng">
        <test-method status="FAIL" signature="base(java.lang.String, java.lang.String)[pri:1, instance:it.pkg.dynamic.Testng@57baeedf]" name="base" duration-ms="0" started-at="2019-11-06T07:36:03Z" data-provider="classname" finished-at="2019-11-06T07:36:03Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void it.pkg.dynamic.Testng.base(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : classname in class org.testng.annotations.DataProvider]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void it.pkg.dynamic.Testng.base(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : classname in class org.testng.annotations.DataProvider
at org.testng.internal.Parameters.findDataProvider(Parameters.java:513)
at org.testng.internal.Parameters.handleParameters(Parameters.java:691)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- base -->
      </class> <!-- it.pkg.dynamic.Testng -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
